.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: Distribution
+on:
+  push:
+    branches-ignore: ["*"]
+    tags: ["*"]
+
+jobs:
+  upload-package:
+    name: "Build & upload package"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags') && github.repository == 'sprockets/sprockets.mixins.sentry'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7  # use 3.7 for wheel compatibility
```

Comment:
This shouldn't matter since the wheel _should_ be tagged as `py3-none-any` in either case.

Reply:
It does indeed. Updated to build on 3.9 in ed7c70518d1a18527cbca257717a10c88608dbd7